packages/story-editor/src/utils/usePageAsCanvas.js

```diff
@@ -98,6 +98,11 @@ function usePageAsCanvas() {
               const promise = htmlToImage.toCanvas(fullbleedContainer, {
                 fontEmbedCss: '',
                 pixelRatio: 1,
+                // exclude selected element from generated image to prevent interfereing with contrast calculation. true includes and false excludes
+                filter: (node) =>
```

Comment:
We could maybe move this into a separate function and only use if if the element to exclude is actually set.

Note that we also have `nodesById` available via `useCanvas`, we could get the node directly by the element ID and use that in the filter for the comparison instead.

Reply:
Good point , using the `selectedElementIds` property from the `useStory` hook was enough because the node that `html-to-image` was giving me was a bit different from the node I was getting from `nodesById(selectedElementIds[0]` so I couldn't compare the two node directly using `isEqualNode` but doing it with just the id from `selectedElementIds` is still sufficient and maybe a little faster